sklearn/metrics/scorer.py

```diff
@@ -66,6 +68,52 @@ def __repr__(self):
                    "" if self._sign > 0 else ", greater_is_better=False",
                    self._factory_args(), kwargs_string))
 
+    def _get_labels_from_estimator(self, estimator, target_type):
+        """Check if estimator has a classes argument and if scorer has a
+        labels argument and return new kwargs for scorer if label not already
+        set in make score initialization"""
+        est_has_classes = hasattr(estimator, "classes_")
```

Comment:
You could also do ``classes = getattr(estimator, "classes_", None)`` here do ``if classes is not None`` later. That would mean getting the attribute only once and saving a variable name ;)